File: attached_assets/Starter_code/Java_starter_code/src/main/java/com/example/insurance/domain/Customer.java
package com.example.insurance.domain;

import jakarta.persistence.*;
import java.util.*;

@Entity
public class Customer {
    @Id @GeneratedValue(strategy = GenerationType.UUID)
    private String id;
    @Column(nullable = false, unique = true)
    private String email;
    @Column(nullable = false)
    private String name;
    private String phone;

    @OneToMany(mappedBy = "customer")
    private List<Policy> policies = new ArrayList<>();

    public String getId() { return id; }
    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }
    public String getName() { return name; }
    public void setName(String name) { this.name = name; }
    public String getPhone() { return phone; }
    public void setPhone(String phone) { this.phone = phone; }
    public List<Policy> getPolicies() { return policies; }
}

File: attached_assets/Starter_code/Dotnet_starter_code/Insurance.Api/Models/Customer.cs
namespace Insurance.Api.Models;

public class Customer
{
    public string Id { get; set; } = Guid.NewGuid().ToString();
    public string Email { get; set; } = default!;
    public string Name { get; set; } = default!;
    public string? Phone { get; set; }
    public List<Policy> Policies { get; set; } = new();
}


